### PR TITLE
SWS-313 Handle graph query params

### PR DESCRIPTION
**Pull request by jshaughn**
- add bookmarkability by honoring graph url query params
- also, fix refresh bug introduced with original time filter PR

### CODE REVIEW

File: src/pages/ServiceGraph/ServiceGraphPage.tsx [hunk 1]
```diff
@@ -27,10 +29,14 @@ export default class ServiceGraphPage extends React.Component<RouteComponentProp
     this.filterChange = this.filterChange.bind(this);
     this.handleError = this.handleError.bind(this);
 
+    const search = routeProps.location.search;
+    const params = new URLSearchParams(search);
+    let graphInterval = params.get('interval');
```
Comment (mtho11) [inline src/pages/ServiceGraph/ServiceGraphPage.tsx]: `const` instead of `let`

**Comment by mtho11** [comment]
@vnugent that is counter to what I have learned my react classes. You should not cause any side-effects in `componentWillMount` and it mainly exists for historical reasons.

**Comment by vnugent** [comment]
we're saying the same thing :) In the PR data is fetched in componentWillMount()
https://github.com/jshaughn/swsui/blob/c73c2a53489f1f19bf3c1642aab07f48043d342e/src/components/GraphFilter/GraphFilter.tsx#L69

File: src/components/GraphFilter/GraphFilter.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ export class GraphFilter extends React.Component<GraphFilterProps, GraphFilterSt
     };
   }
 
-  componentWillMount() {
+  componentDidMount() {
```
Comment (mtho11) [inline src/components/GraphFilter/GraphFilter.tsx]: In the future should also do a `.catch()` for failures

**Pull request merged**